kubernetes/delete-weblogic-domain-resources.sh

```diff
@@ -57,33 +57,11 @@ cat << EOF
               max-seconds is exceeded and there are any leftover
               kubernetes resources.
 
-    Phase 5:  Delete any singleton Voyager load balancer resources
```

Comment:
Since this change removes the helm delete, the 'Phase 3' in the comment should be removed.